src/oop/special_class_method/hard/q6/QuestionText.md

```diff
@@ -0,0 +1,38 @@
+# oop-special_class_method-hard-q6
+
+## Question Text
+
+You are writing a Java program that mimics different Vehicles and their attributes. Write a Java class called **Car**
```

Comment:
This paragraph has a lot of information made redundant by the specifications below.